lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -66,6 +67,7 @@
 
   box-sizing: border-box;
   flex: 1 0 $nx-content-width;
+  min-width: 0;
```

Comment:
Added to mitigate the `overflow` change below.  Without this, long unbroken text content would make the `nx-page-main` exceed its proper width and smoosh the sidebar